Doc/library/collections.rst

```diff
@@ -618,19 +619,19 @@ added elements by appending to the right and popping to the left::
             d.append(elem)
             yield s / n
 
-The :meth:`rotate` method provides a way to implement :class:`deque` slicing and
-deletion.  For example, a pure Python implementation of ``del d[n]`` relies on
-the :meth:`rotate` method to position elements to be popped::
+The :meth:`~deque.rotate` method provides a way to implement :class:`deque` slicing
+and deletion.  For example, a pure Python implementation of ``del d[n]`` relies on
+the :meth:`~deque.rotate` method to position elements to be popped::
```

Comment:
Avoid using the same link multiple times in one paragraph. ` ``rotate()`` ` would look better.